integration/common/openlineage/common/provider/redshift_data.py

```diff
@@ -0,0 +1,161 @@
+# Copyright 2018-2022 contributors to the OpenLineage project
+# SPDX-License-Identifier: Apache-2.0
+
+import logging
+import attr
+
+from typing import Any, Optional, Dict, List
+import botocore
+
+from openlineage.common.dataset import Dataset, Source
+from openlineage.common.models import DbTableSchema, DbColumn
+from openlineage.common.schema import GITHUB_LOCATION
+from openlineage.common.sql import DbTableMeta
+from openlineage.client.facet import BaseFacet, ErrorMessageRunFacet
+import traceback
+
+
+@attr.s
+class RedshiftDataRunFacet(BaseFacet):
+    """
+    Facet that represents relevant statistics of redshift run.
+    :param duration: duration of the run in nanoseconds
+    :param resultRows: number of rows in the result
+    :param resultSize: size of the result in bytes
+    """
+
+    duration: Optional[int] = attr.ib(default=None)
+    resultRows: Optional[int] = attr.ib(default=None)
+    resultSize: Optional[int] = attr.ib(default=None)
```

Comment:
We should use standard facet: https://github.com/OpenLineage/OpenLineage/blob/main/spec/facets/OutputStatisticsOutputDatasetFacet.json

Reply:
Should we get rid of duration stat or add it as optional in proposed facet?

Reply:
@JDarDagran We can remove it for now